docs/pages/access-controls/access-requests/resource-requests.mdx

```diff
@@ -323,52 +340,169 @@ spec:
     max_session_ttl: 1h
 ```
 
+#### `kube_cluster`
+
+You can restrict access to searching `kube_cluster` resources by assigning
+values to the `kubernetes_labels` field in the `spec.allow` or `spec.deny`
+fields. 
+
+The following role allows access to Kubernetes clusters with the `env:staging`
+label:
+
+```yaml
+kind: role
+metadata:
+  name: kube-access
+version: v6
+spec:
+  allow:
+    kubernetes_labels:
+      'env': 'staging'
+    kubernetes_resources:
+      - kind: pod
+        namespace: "*"
+        name: "*"
+  deny: {}
+```
+
+#### `pod`
+
+You can restrict access to creating `pod` resources by assigning values to the
+`kubernetes_resources` field in the `spec.allow` or `spec.deny` fields. 
+
+The following role allows access to Kubernetes pods with the name `nginx` in any
+namespace, and all pods in the `dev` namespace:
+
+```yaml
+kind: role
+metadata:
+  name: kube-access
+version: v6
+spec:
+  allow:
+    kubernetes_resources:
+      - kind: pod
+        namespace: "*"
+        name: "nginx*"
+      - kind: pod
+        namespace: "dev"
+        name: "*"
+    kubernetes_groups:
+      - viewers
+  deny: {}
+```
+
+##### Searching for pod resources
```

Comment:
Searching will be introduced soon but currently, we don't support it.

Reply:
I've renamed this section so we don't imply we support `tsh request search --kind=pods`